server/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM --platform=linux/amd64 alpine:latest
+
+# Update package list and install required certificates
+RUN apk --no-cache add ca-certificates \
+    && apk add --no-cache libc6-compat
+
+# Set a working directory (optional)
+WORKDIR /app
+
+# Copy the local binary into the container
+COPY ./deweb-server_linux_amd64 /app/deweb-server_linux_amd64
```

Comment:
I'm unsure we should do that
IMO, we should divide this dockerfile into 2 stages, one to build and the other to run the image, so we don't require the host computer to have already built the binary

Reply:
Yes, that's a good solution; it's called a multi-stage build, which we discussed last week. 

Another option is to place the binary directly into the image (as done here) and tag the image based on the binary (built by a CI or a third party).

Ex : deweb_server:1.0.0 contains the version 1.0.0 of deweb ... and user choose different version of the docker image based on the deweb server. (And pass a config file if needed). 

But yeah let me work on multi stage build if u want